FreeRTOS/FreeRTOS-Plus-TCP
# IPv6/Multi: Remove unused macro from FreeRTOS_TCP_IP.c

Description
-----------
The macro `ipconfigTCP_ACK_EARLIER_PACKET` in FreeRTOS_TCP_IP.c was once added as a testing option. By default it is defined as 1, and that seems to work well for everyone. So now it is time to remove it, just like we did in the earlier PR #289. In that PR you will find more information about the macro.

Test Steps
-----------
I passed the regular tests with TCP, such as IPERF.

Related Issue
-----------
This PR doesn't have the intention to solve an issue.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
